### PR TITLE
Remove sharing buttons override styles on twentynineteen and twentysixteen themes

### PR DESCRIPTION
**Pull request by davemart-in**
## Proposed changes:

This PR extends the work in #28874 to update the look and feel of the sharing icons. You can see a full list of updates that need to be made [here](https://github.com/Automattic/jetpack/issues/27817).

Previously there had been a border and a box-shadow. I suspect that whomever added these overrides disliked how chunky it looked with both a border and a box-shadow so they decided to override the styles and remove the box-shadow.  In #28874 I remove the border, which left the sharing buttons on these two themes without any sort of visual border. I've fixed this by removing these override styles for both themes.

### Before

twentynineteen-before

### After

twentynineteen-after

### Before

twentysixteen-before

### After

twentysixteen-after

## Jetpack product discussion
- #27817
- pe7F0s-i3-p2
- pe7F0s-qg-p2

## Does this pull request change what data or activity we track or use?
Nope.

## Testing instructions:
- Follow instructions to [set up JP locally](https://github.com/Automattic/jetpack/blob/trunk/docs/quick-start.md) (with jurassic.tube proxy), or use jurassic ninja
- Ensure that you have sharing buttons activated
- Ensure that you have at least one published blog post
- Set twentynineteen and then twentysixteen as your theme
- Visit the blog post

## Related
- Parent PR -> #28874
- #27817
- https://github.com/Automattic/wp-calypso/pull/73126

### CODE REVIEW

**Review by jeherve — COMMENTED** [review]
👍 This should be good to go once the sharing changes are merged. Good catch!

**Comment by davemart-in** [comment]
Just rebased this PR.

**Review by jeherve — APPROVED** [review]

**Pull request merged**
